### DISCUSSION TITLE
Part number for the picozed heatsink

### DISCUSSION THREAD

**Discussion by elsevers** [Severson-Group/AMDC-Hardware#238] [Q&A] [answered]
Hello everyone, it has become standard for us to use heatsinks on the picozed. Can someone share a partnumber for what they have been using? (I am not seeing it on our BOM or other docs)

**Comment by npetersen2** [comment, marked as answer]
![image](https://github.com/Severson-Group/AMDC-Hardware/assets/20168990/9b4eb451-d5dd-4065-9a0d-7b530bf5daa2)

**Reply by elsevers** [reply]
Okay, I somehow missed the obvious. Thank you!
